control/velocity_controller_lqr/velocity_controller_lqr/velocity_controller_lqr_lib.py

```diff
@@ -0,0 +1,289 @@
+#!/usr/bin/env python3
+import math as math
+
+import numpy as np
+
+import control as ct
+
+
+class LQRController:
+    def __init__(
+        self,
+        q_surge,
+        q_pitch,
+        q_yaw,
+        r_surge,
+        r_pitch,
+        r_yaw,
+        i_surge,
+        i_pitch,
+        i_yaw,
+        max_force,
+    ):
+        self.integral_error_surge = 0.0
+        self.integral_error_pitch = 0.0
+        self.integral_error_yaw = 0.0
+
+        self.surge_windup = False  # Windup variable
+        self.pitch_windup = False  # Windup variable
+        self.yaw_windup = False  # Windup variable
+
+        self.max_force = max_force  # Maximum force that can be applied
+
+        self.i_surge = i_surge  # Integral gain for surge
+        self.i_pitch = i_pitch  # Integral gain for pitch
+        self.i_yaw = i_yaw  # Integral gain for yaw
+
+        self.q_surge = q_surge  # Surge cost
+        self.q_pitch = q_pitch  # Pitch cost
+        self.q_yaw = q_yaw  # Yaw cost
+
+        self.r_surge = r_surge  # Surge control cost
+        self.r_pitch = r_pitch  # Pitch control cost
+        self.r_yaw = r_yaw  # Yaw control cost
+
+        # VARIABLES
+        self.inertia_matrix_inv = np.linalg.inv(
+            np.array([[30, 0.6, 0], [0.6, 1.629, 0], [0, 0, 1.769]])
```

Comment:
Could the values for the inertia matrix be imported from orca.yaml, instead of being hard coded in?